sites/all/modules/_custom/forms/forms.module

```diff
@@ -298,6 +298,10 @@ function forms_form_alter(&$form, &$form_state, $form_id){
 	}
 }
 
+function forms_init(){
+	drupal_add_js(drupal_get_path('module', 'forms').'/static/nc_common_functions.js');
+}
```

Comment:
how about using this 
```
$form['#attached']['js'] = array(
  drupal_get_path('module', 'ajax_example') . '/ajax_example.js',
);
```
this I feel is a cleaner way to attach js. The init functions have peculiar and fixed timing of being called on the page load.

Reply:
https://www.drupal.org/docs/7/api/javascript-api/adding-javascript-to-your-theme-or-module

Link in case you want to read more.